### PR TITLE
Document external domains we need access to

### PR DESCRIPTION
**Pull request by glitchcrab**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/reference/domain-whitelist/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Below is a list of the external domains we require access to for our clusters to function.
+
+- flatcar.com
+    - *.flatcar.com
+- quay.io
+    - *.quay.io
+- github.com
+    - *.github.com
+- release.core-os.net
+    - *.release.core-os.net
```
Comment (QuentinBisson) [inline src/content/reference/domain-whitelist/index.md:26]: Why do we need this one?

Comment (glitchcrab) [reply src/content/reference/domain-whitelist/index.md:26]: hmm that's a good point - i'm not sure we do. i was just dumping a list from a github issue. i'll remove it.

**Review by QuentinBisson — APPROVED** [review]
This makes sense to me but could we get another set of eyes?

File: src/content/reference/domain-whitelist/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - https://github.com/orgs/giantswarm/teams/se
+---
+
+# External Domains
+
+Below is a list of the external domains we require access to for our clusters to function.
+
+- flatcar.com
+    - *.flatcar.com
+- quay.io
```
Comment (QuentinBisson) [inline src/content/reference/domain-whitelist/index.md:21]: btw, we are missing opsgenie :D

**Review by marians — CHANGES_REQUESTED** [review]
Please do not use the term "whitelist", as it's loaded with connotations. We are planning to keep up with the Kubernetes project on this sort of terminology issues. See https://github.com/giantswarm/giantswarm/issues/14964#issuecomment-819428368

File: src/content/reference/domain-whitelist/index.md (hunk 2 above)
Comment (glitchcrab) [reply src/content/reference/domain-whitelist/index.md]: also fixed!

**Review by marians — APPROVED** [review]
Thanks for changing the naming!

Wanna wrap these literals in single backticks?

File: src/content/reference/domain-allowlist/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+menu:
+  main:
+    parent: reference
+user_questions:
+  - What domains do Giant Swarm clusters need access to?
+owner:
+  - https://github.com/orgs/giantswarm/teams/se
+---
+
+# External Domains
```
Comment (marians) [inline src/content/reference/domain-allowlist/index.md:15]:
```suggestion
# External domains Giant Swarm clusters need access to
```

To make the headline more speaking, also use sentence case

**Review by pipo02mix — COMMENTED** [review]
Do we need NTP server access too?

**Comment by glitchcrab** [comment]
thanks for the kick - i forgot this PR

**Pull request merged**